### DISCUSSION TITLE
Wondering Schematic Illustration Tool for SpinalHDL Documents

### DISCUSSION THREAD

**Discussion by hilaolu** [SpinalHDL/SpinalHDL#1405] [Q&A]
Hi, using SpinalHDL is a wonderful experience for me. I found the illustations for SpinalHDL block design is clear and sound, I am finding an illustration to draw such schematic illustrations. Any suggestions are welcomed, many thanks.

**Comment by Readon** [comment]
How about diagram gen by #1079

**Comment by andreasWallner** [comment]
I you mean some of the drawings from the documentation, those are hand-drawn with a vector graphics editor like inkscape.
